src/views/ResultsPage/ResultsPage.jsx

```diff
@@ -600,6 +603,7 @@ const ResultsPage = () => {
 			</div>
 		);
 	};
+	const pagerExists = trialResults != null && trialResults.total / 10 > 1;
```

Comment:
How is the denominator value derived (10)?

Reply:
Since we do 10 results per page, that's how I derived it

Reply:
so I used the value to determine how many pages the pager has listed